.github/workflows/tsbs-test.yaml

```diff
@@ -0,0 +1,79 @@
+name: tsbs-test
+on:
+  pull_request:
+    branches:
+       - main
+jobs:
+  tsbs-test:
+    name: tsbs-test
+    env:
+      DB_URL: ${{ secrets.PERFORMANCE_DB }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          pwd
+          git status
+          git fetch
+          sudo apt install tree
+          cd ~
+          tree -d
+      - run: sudo cat /proc/cpuinfo
+      - run: sudo cat /proc/meminfo
+      - run: |
+          sudo apt update
+          sudo apt upgrade
+          sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+          wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt update
+          sudo pip install psycopg2
+          sudo pip install pandas
+          sudo pip install matplotlib
+          sudo pip install numpy
+          sudo pip install jq
+          sudo apt install libssl-dev libkrb5-dev cmake make
+          sudo apt install postgresql-14 postgresql-server-dev-14
+          git fetch --all --tags
+          echo y | ./bootstrap -DCMAKE_BUILD_TYPE=Debug -DASSERTIONS=ON
```

Comment:
If you have assertions on, it could affect performance. Might be better to do a Release build for performance benchmarks.